targets/jm-relay-1.0/profile/relay.c

```diff
@@ -14,7 +14,8 @@ static const relay_params_t params = {
     .pin_relay_feedback = RELAY_FEEDBACK,
     .pin_relay_led = RELAY_LED,
     .drive_active_lo = 0,
-    .led_active_lo = 1
+    .led_active_lo = 1,
+    .initial_state = false
```

Comment:
the struct is 0-initialized by default, so you can skip the initial_state=false, but you can also leave it for clarity